lib/config/sass-lint.yml

```diff
@@ -35,6 +35,7 @@ rules:
   no-url-protocols: 1
   no-vendor-prefixes: 1
   no-warn: 1
+  no-trailing-whitespace: 1
```

Comment:
Just a small one but you could you put this in alphabetical order with the rest of the disallow rules so as to follow the pattern. Should be above `no-trailing-zero'.